test/videojs.ads.snapshot.js

```diff
@@ -1,467 +1,365 @@
-  ok(!playCalled, 'content playback should not resume');
+
+  this.player.trigger('ended');
+  playSpy.reset();
+
+  // trigger a lot of post-rolls
+  this.player.ads.startLinearAdMode();
+  this.player.src('http://example.com/ad1.mp4');
+  this.player.trigger('loadstart');
+  this.player.src('http://example.com/ad2.mp4');
+  this.player.trigger('loadstart');
+  this.player.ads.endLinearAdMode();
+  this.player.trigger('playing');
+  this.player.trigger('ended');
+  assert.strictEqual(this.player.ads.state, 'content-playback', 'Player should be in content-playback state after a post-roll');
+  assert.strictEqual(this.contentPlaybackSpy.callCount, 2, 'A content-playback event should have been triggered');
+  assert.strictEqual(this.contentPlaybackReason(1), 'ended', 'The reason for content-playback should have been ended');
+  assert.notOk(playSpy.called, 'content playback should not resume');
 });
 
-// "ended" does not fire when the end of a video is seeked to directly
-// in iOS 8.1
-test('does resume playback after postrolls if "ended" does not fire naturally', function() {
-  var playCalled = false, callbacks = [], i;
-  player.src('http://media.w3.org/2010/05/sintel/trailer.mp4');
+// "ended" does not fire when the end of a video is seeked to directly in iOS 8.1
+QUnit.test('does resume playback after postrolls if "ended" does not fire naturally', function(assert) {
+  assert.expect(2);
 
   // play the video
-  player.trigger('loadstart');
-  player.trigger('adsready');
-  player.trigger('play');
-  player.trigger('adtimeout');
+  this.player.src('http://media.w3.org/2010/05/sintel/trailer.mp4');
+  this.player.trigger('loadstart');
+  this.player.trigger('adsready');
+  this.player.tech.trigger('play');
+  this.player.trigger('adtimeout');
 
   // finish the video and watch for play()
-  player.ended = function() {
+  this.player.ended = function() {
     return true;
   };
-  player.trigger('ended');
+
+  this.player.trigger('ended');
+
   // play a postroll
-  player.ads.startLinearAdMode();
-  player.src('http://example.com/ad1.mp4');
-  player.ads.endLinearAdMode();
+  this.player.ads.startLinearAdMode();
+  this.player.src('http://example.com/ad1.mp4');
+  this.player.ads.endLinearAdMode();
 
   // reload the content video while capturing timeouts
-  window.setTimeout = function(callback) {
-    callbacks.push(callback);
-  };
-  player.trigger('contentcanplay');
+  var setTimeoutSpy = sinon.spy(window, 'setTimeout');
+  this.player.trigger('contentcanplay');
+  assert.strictEqual(setTimeoutSpy.callCount, 1, 'set a timeout to check for "ended"');
 
-  ok(callbacks.length > 0, 'set a timeout to check for "ended"');
   // trigger any registered timeouts
-  player.play = function() {
-    playCalled = true;
-  };
-  i = callbacks.length;
-  while (i--) {
-    callbacks[i]();
-  }
-  ok(playCalled, 'called play() to trigger an "ended"');
+  var playSpy = sinon.spy(this.player, 'play');
+  this.clock.tick(10000);
```

Comment:
Looked again and it's 1250: 20 `tryToResume` timeouts at 50ms each + `resumeEndedTimeout` at 250ms.